### PR TITLE
Update iframe delegation section

### PR DESCRIPTION
**Pull request by beaufortfrancois**
R: @mounirlamouri

### CODE REVIEW

File: src/content/en/updates/2017/09/autoplay-policy-changes.md [hunk 1]
```diff
@@ -92,7 +92,11 @@ permission, it can delegate that permission to cross-origin iframes with a new
 same-origin iframes.
 
 <pre class="prettyprint">
-&lt;iframe src="myvideo.html" allow="autoplay">
+&lt;!-- Autoplay is allowed. -->
+&lt;iframe src="https://cross-origin.com/myvideo.html" allow="autoplay">
+
+&lt;!-- Autoplay and Fullscreen are allowed. -->
+&lt;iframe src="https://cross-origin.com/myvideo.html" allow="autoplay;fullscreen">
```
Comment (mounirlamouri) [inline src/content/en/updates/2017/09/autoplay-policy-changes.md]: nit: `allow="autoplay; fullscreen"` (one space would make things more readable)

Comment (beaufortfrancois) [reply src/content/en/updates/2017/09/autoplay-policy-changes.md]: Done.

**Comment by beaufortfrancois** [comment]
image

File: src/content/en/updates/2017/09/autoplay-policy-changes.md [hunk 2]
```diff
@@ -92,16 +92,23 @@ permission, it can delegate that permission to cross-origin iframes with a new
...
 
 When the feature policy for autoplay is disabled, calls to `play()` without a
 user gesture will reject the promise with a `NotAllowedError` DOMException. And
 the autoplay attribute will also be ignored.
 
-Note: You can try out this feature policy by enabling the experimental flag
+You can try out this feature policy by enabling the experimental flag
 `chrome://flags/#enable-experimental-web-platform-features`.
 
+Warning: Contrary to what was announced previously, `<iframe gesture=media>` is
```
Comment (johnpallett) [inline src/content/en/updates/2017/09/autoplay-policy-changes.md]: We never said <iframe gesture=media> was supported. Instead:

Warning: Older articles incorrectly recommend using the attribute _gesture=media_ which is not supported.

**Comment by johnpallett** [comment]
Added comment in notes

**Comment by beaufortfrancois** [comment]
Thank you @johnpallett.

@petele Can you merge?

**Comment by beaufortfrancois** [comment]
It is now merged with https://github.com/google/WebFundamentals/pull/5760.

**Pull request closed without merging**